variables.tf

```diff
@@ -26,6 +26,12 @@ variable "public_access_enabled" {
   default     = false
 }
 
+variable "private_endpoint" {
+  type        = bool
+  description = "To enable/disable the private endpoint in the account public access."
```

Comment:
Set to true to use the private IAM endpoint which is used to configure the public access setting

Reply:
Done.

Reply:
This does not match what I said in my comment. I would also default it to back to `false`

Reply:
Sorry my bad. Updated the description and changed the default value to `false`.